.vscode/settings.json

```diff
@@ -2,7 +2,6 @@
 {
     "files.exclude": {
         "out": true,
-        "node_modules": true,
```

Comment:
Since the webview's artifacts are used in the release script, I would like to keep an eye on the node_module dir. Otherwise, I won't be able to see node_modules in VScode IDE